rotkehlchen/api/v1/schemas.py

```diff
@@ -3609,6 +3609,7 @@ class CalendarCommonEntrySchema(AnyBlockchainAddress):
     name = fields.String(required=True)
     description = fields.String(load_default=None)
     counterparty = EvmCounterpartyField()
+    color = fields.String(load_default=None)
```

Comment:
We already have a color field. Please use that. Search for `ColorField` in this file.